homeworks/zsoltcsengeri/2_vars_datatypes/user_info.py

```diff
@@ -0,0 +1,81 @@
+from pprint import pprint
+
+# The dictionary
+user_info = {
+    "name": "Mike",
+    "age": 25,
+    "favourite_meals": ["pizza", "carbonara", "sushi"],
+    "phone_contacts": {
+        "Mary": "+36701234567",
+        "Tim": "+36207654321",
+        "Tim2": "+36304567321",
+        "Jim": "+364005000",
+    },
+}
+
+
+# 1 Request 4 programming languages from the user (comma-separated)
+programming_languages = input(
+    "Please give me 4 programming languages comma-separated: "
+)
+
+print(type(programming_languages))
+
+# Convert input string to a list. Another method to convert to list is to use list() function:list(programming_languages)
+programming_languages_list = programming_languages.split(",")
+print(programming_languages_list)
+print(type(programming_languages_list))
+
+# Add 'skills' key to user_info dictionary
+user_info["skills"] = programming_languages_list
```

Comment:
nem hiba, csak egy tipp: mivel a programming_langueages_list-et nem használod később, az is jó, ha csak simán így adod meg a skillst:
user_info["skills"] = programming_languages.split(",")